collections/_article/godot-arrives-in-the-epic-games-store.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Godot Arrives in the Epic Games Store"
+excerpt: "We are honored to announce that Epic Games is offering Godot Engine a place on its own digital distribution platform, the Epic Games Store."
+categories: ["news"]
+author: Emi
+image: /storage/blog/covers/godot-epic-games-store.jpg
+date: 2023-03-17 14:00:00
+---
+
+We are honored to announce that Epic Games is offering Godot Engine a place on its own digital distribution platform, the Epic Games Store. Starting today, you can choose to use EGS to [download the engine](https://store.epicgames.com/p/godot-engine) and keep it up to date with every release.
```

Comment:
The first sentence is a bit awkward. It almost sounds like we are saying that Epic Games has offered Godot a spot on _Godot's_ store. I would simplify to remove the pronoun
```suggestion
We are honored to announce that Epic Games is offering Godot Engine a place on the Epic Games Store. Starting today, you can choose to use EGS to [download the engine](https://store.epicgames.com/p/godot-engine) and keep it up to date with every release.
```

Or alternatively, if you want to stay closer to the current language
```
We are honored to announce that Epic Games is offering Godot Engine a place on Epic Game's own digital distribution platform, the Epic Games Store. Starting today, you can choose to use EGS to [download the engine](https://store.epicgames.com/p/godot-engine) and keep it up to date with every release.
```

Reply:
I'd go with the alternative, but would reduce the second mention to just Epic instead of Epic Games. Less repetition and still valid.